### PR TITLE
feat(customersupport): open customer support

### PR DESCRIPTION
**Pull request by mt-neus**
Api to open customer support (intercom) from Vault with tests

### CODE REVIEW

**Review by mt-micky — DISMISSED** [review]

**Comment by mt-neus** [comment]
Not merging until we release in staging

**Comment by mt-roberto** [comment]
- [ ] Update README

**Comment by mt-neus** [comment]
> * [x]  Update README

done

File: README.md [hunk 1]
```diff
@@ -50,14 +50,14 @@ Config properties:
...
-  appToken, // string; // [optional] The Access Token granted through oauth
-  isTestEnvironment // boolean; // [optional] If you wanna use the staging or production environemnt
+    responseType, // string; // Tells the authorization server which grant to execute (e.g.: code).
+    scope, // string[]; // A list of permissions that the application requires.
+    redirectUri, // string; // URI to return the user to after authorization is complete.
+    continueTo, // string; // [optional] Parameter appended as `continue` to the `redirectUri`.
+    locale, // string; // [optional] To force the display to a specific language (e.g.: en-AU).
+    state, // string; // [optional] An opaque value, used for security purposes. If this request parameter is set in the request, then it is returned to the application as part of the redirect_uri.
+    appToken, // string; // [optional] The Access Token granted through oauth
+    isTestEnvironment; // boolean; // [optional] If you wanna use the staging or production environemnt
```
Comment (mt-roberto) [inline README.md]: Extra-space 🌌

**Comment by mt-kenny** [comment]
Going to close this ticket since this was fulfilled by the deep link feature (#45 ).

**Pull request closed without merging**